credits/Citing-OSCAR.md

```diff
@@ -22,11 +22,10 @@ If you are using **BibTeX**, you can use the following BibTeX entries:
 @misc{OSCAR,
   key          = {OSCAR},
   organization = {The OSCAR Team},
-  title        = {OSCAR -- Open Source Computer Algebra Research system,
-                  Version 1.0.0},
-  year         = {2024},
+  title        = {OSCAR -- Open Source Computer Algebra Research system, Version {{ site.data.oscar_version.latest_release }}},
+  year         = { {{ site.data.oscar_version.latest_release_date | split: " " | last }}},
```

Comment:
Try this
```suggestion
  year         = { {{- site.data.oscar_version.latest_release_date | split: " " | last }}},
```
For background, see https://shopify.github.io/liquid/basics/whitespace/

Reply:
Seems to have worked :-)

Reply:
Indeed. This works. Thx!